connectors/airwatch/pom.xml

```diff
@@ -160,6 +161,87 @@
+                                                <files>
+                                                    <file>
+                                                        <source>${project.basedir}/src/main/resources/application.yml</source>
+                                                        <outputDirectory>./</outputDirectory>
+                                                    </file>
+                                                    <file>
+                                                        <source>${project.basedir}/src/test/resources/docker/managed-apps.yml</source>
+                                                        <outputDirectory>./</outputDirectory>
+                                                    </file>
+                                                    <file>
+                                                        <source>${project.basedir}/src/test/resources/docker/application.properties</source>
+                                                        <outputDirectory>./</outputDirectory>
+                                                    </file>
+
+                                                </files>
+                                            </inline>
+                                            <targetDir>/opt/vmware/airwatch-connector</targetDir>
+                                        </assembly>
+                                        <from>openjdk:8u151-jre</from>
+                                        <workdir>/opt/vmware/airwatch-connector</workdir>
+                                        <entryPoint>
+                                            <shell>
+                                                java -server -XX:+UnlockExperimentalVMOptions \
+                                                 -Xss256k \
+                                                 -XX:+UseCGroupMemoryLimitForHeap \
+                                                 -XX:MaxRAMFraction=1 \
+                                                 -XX:+HeapDumpOnOutOfMemoryError \
+                                                 -XX:HeapDumpPath=/tmp \
+                                                 -jar /opt/vmware/airwatch-connector/airwatch-connector-${project.version}.jar \
+                                                 --logging.level.com.vmware=${container.logging.level} \
+                                                 --spring.profiles.active=prod,external \
+                                                 --tenant.publicKeyUrl="https://herocard.vmwareidentity.com/SAAS/API/1.0/REST/auth/token?attribute=publicKey&amp;format=pem" \
+                                                 --security.oauth2.resource.jwt.key-uri="https://herocard.vmwareidentity.com/SAAS/API/1.0/REST/auth/token?attribute=publicKey&amp;format=pem" \
+                                                 --spring.config.location=/opt/vmware/airwatch-connector/managed-apps.yml,/opt/vmware/airwatch-connector/application.properties  \
```

Comment:
There's no point burying config in the container. The user needs control over this.

Reply:
Removed all the configs.